handlers/homepage/handler.go

```diff
@@ -2,9 +2,10 @@
 package homepage
 
 import (
-	"github.com/ONSdigital/dp-frontend-renderer/config"
 	"net/http"
 
+	"github.com/ONSdigital/dp-frontend-renderer/config"
+
```

Comment:
Remove this space and then the two `github.com/ONSdigital/dp-frontend-renderer/…` lines will be together after `go fmt`